subsys/bluetooth/controller/Kconfig

```diff
@@ -584,5 +584,13 @@ config BT_CTLR_SDC_CS_COUNT
 	help
 	  Set the number of concurrent connections that can support Channel Sounding procedure.
 
+config BT_CTLR_SDC_LE_POWER_CLASS_1
+	bool "Device supports transmitting at LE Power Class 1 level"
+	default y if BT_CTLR_TX_PWR_ANTENNA >= 10
```

Comment:
Should this be a `select` so that the Kconfig is enforced with tx power > 10dBm? or is this a `should` in the specification?

Reply:
> Should this be a `select` so that the Kconfig is enforced with tx power > 10dBm? or is this a `should` in the specification?

The spec doesn't really give much information at all on this as far as I can see, I think this is fine as is using default.

Reply:
> Don't know what this actually does or if a user would want to override it but the way it works here (which I assume is wrong) is it will default to the value as per the condition, if the user changes the Kconfigs in the condition, the existing value will stay, despite maybe the conditions meaning that the value is no longer correct

I'm not sure I follow, do you think this is wrong as it is currently?

Reply:
This kconfig only sets the static default right? The application could set the txpower to >10 dbm dynamically even if `BT_CTLR_TX_PWR_ANTENNA < 10` so the kconfig should be application-controllable.